teams/project-trait-system-refactor.toml

```diff
@@ -0,0 +1,26 @@
+name = "project-trait-system-refactor"
+kind = "project-group"
+# Really should be types, but types is already a subteam,
+# so that's not allowed for now.
```

Comment:
@pietroalbini do you know why this restriction was put in place?

Reply:
maybe because we of the governance webpage as there's currently a 1 to 1 correspondence between the parent team and the governance page: https://www.rust-lang.org/governance/teams/compiler :thinking: